data/ReferrerWhitelist.json

```diff
@@ -4,7 +4,9 @@
             "<all_urls>": [
                 "https://use.typekit.net/*",
                 "https://api.geetest.com/*",
-                "https://cloud.typography.com/*"
+		"https://cloud.typography.com/*",
+		"https://*.vimeocdn.com/*",
+		"https://player.vimeo.com/*"
```

Comment:
could you please indent using spaces instead of tabs here?

Reply:
sure. Done.